<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WallThicknessCalculationsClipperViewControls</class>
 <widget class="QWidget" name="WallThicknessCalculationsClipperViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>95</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QmitkTemplate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_1">
     <item>
      <widget class="QPushButton" name="button_1">
       <property name="text">
        <string>Find Centrelines</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_2">
       <property name="text">
        <string>Display Clippers</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_4">
       <property name="text">
        <string>Clip Seg Image</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QVTKOpenGLWidget" name="widget_1" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">border-width: 1px;
border-color: black;
border-radius: 10px;
background-color: yellow;</string>
       </property>
       <property name="text">
        <string> Automatic </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="slider">
       <property name="maximum">
        <number>2</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinBox">
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>20.00000000000000